examples/speculative_decoding/launch_train.sh

```diff
@@ -87,7 +91,7 @@ set -x
 # Get the default value for save_steps based on the available number of GPUs
 GPU_COUNT=$(python -c "import torch; print(torch.cuda.device_count())")
 # Calculate save_steps
-DEFAULT_SAVE_STEPS=$((8192 / GPU_COUNT))
+DEFAULT_SAVE_STEPS=$((192 / GPU_COUNT))
```

Comment:
Is this a little too frequent? With 8 GPUS it will save checkpoint every 24 steps

Reply:
That was my mistake, please use `launch_train.sh` and remove `launch.sh`. Thanks!